AdminWebsite/AdminWebsite/ClientApp/src/app/common/model/participant-details.model.ts

```diff
@@ -77,7 +77,8 @@ export class ParticipantDetailsModel {
     showCaseRole(): boolean {
         return this.CaseRoleName.toLowerCase() === CaseRoles.NONE.toLowerCase() ||
             this.CaseRoleName.toLowerCase() === CaseRoles.OBSERVER.toLowerCase() ||
-            this.CaseRoleName.toLowerCase() === CaseRoles.PANEL_MEMBER.toLowerCase()
+            this.CaseRoleName.toLowerCase() === CaseRoles.PANEL_MEMBER.toLowerCase() ||
```

Comment:
as there are some logic involved, could we add some unit tests?

Reply:
thats done